packages/commerce/src/machines/commerce/utils.ts

```diff
@@ -126,24 +101,28 @@ export const getPriceParams = (machineContext: CommerceMachineContext) => {
   }
   const {site, id: sellable_id, type} = sellable
 
+  const upgradeParams = {
+    upgrade_from_sellable_id: upgradeFromSellable?.slug,
+    upgrade_from_sellable: upgradeFromSellable?.type,
+  }
+
+  const sellableStuff: EggheadSellableParam = {
```

Comment:
added some types here and left out the `pickBy` in favor of "assembling" the params with spreads.